de-lib/src/main/java/org/iplantc/de/client/models/collaborators/OldCollaborator.java

```diff
@@ -9,9 +9,9 @@
 
 /**
  * @author sriram
- * 
+ *
  */
-public interface Collaborator extends HasId {
+public interface OldCollaborator extends HasId {
```

Comment:
I might be missing something but is there a reason to keep the `OldCollaborator` around ?